tests/admin_views/admin.py

```diff
@@ -429,7 +430,8 @@ class PostAdmin(admin.ModelAdmin):
     list_display = ['title', 'public']
     readonly_fields = (
         'posted', 'awesomeness_level', 'coolness', 'value',
-        'multiline', 'multiline_html', lambda obj: "foo"
+        'multiline', 'multiline_html', 'multiline_html_allow_tags',
```

Comment:
instead of sorting these, I'd just put 'multiline_html_allow_tags' on the next newline so that we don't need to modify more than one line when removing the deprecated feature